shell_integration/windows/OCContextMenu/OCContextMenu.cpp

```diff
@@ -182,20 +182,38 @@ IFACEMETHODIMP OCContextMenu::InvokeCommand(LPCMINVOKECOMMANDINFO pici)
 {
     std::wstring command;
 
+    CMINVOKECOMMANDINFOEX *piciEx = nullptr;
+    if (pici->cbSize == sizeof(CMINVOKECOMMANDINFOEX))
```

Comment:
maybe could be a >= check instead in case Microsoft introduces an EX2 structure at a later point?